docs/classic/installation-noble.rst

```diff
@@ -0,0 +1,326 @@
+
+WLAN
+====
+
+You should be able to check the WLAN interface (using ``ip link`` for instance):
+
+.. code-block:: bash
+
+    $ ip link show
+    1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+        link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    2: wlP1p1s0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
+        link/ether 90:e8:68:bc:88:a9 brd ff:ff:ff:ff:ff:ff
+
+XXX TODO check compatibility
+Ubuntu server configuration
+===========================
+
+Ubuntu server comes with `netplan and systemd-networkd preinstalled`_. The initial netplan configuration in the image should already take care of the Ethernet interface. To setup a WLAN connection, you can perform the following steps to add the related configuration:
+
+.. code-block:: bash
+
+    # Replace <SSID> with your SSID
+    SSID='<SSID>'
+    # Replace <PASSWD> with your password
+    PASSWD='<PASSWD>'
+    # Create a netplan configuration for the WLAN
+    cat <<EOF | sudo tee /etc/netplan/51-wireless.yaml
+    network:
+      version: 2
+      wifis:
+        wlP1p1s0:
+          dhcp4: yes
+          dhcp6: yes
+          access-points:
+            "$SSID":
+              password: "$PASSWD"
+    EOF
+    sudo netplan apply
+
+Once applied, your network interface should get up and running after a few seconds, which you can confirm using the ``ip address`` command.
+
+.. _netplan and systemd-networkd preinstalled: https://ubuntu.com/blog/a-declarative-approach-to-linux-networking-with-netplan
+
+Install NVIDIA proprietary software
+===================================
+
+The Ubuntu image brings anything necessary to boot Linux on a Jetson development kit. However, to unlock the features of the Tegra SoC (wireless network, bluetooth, GPU, …) you can install additional NVIDIA proprietary drivers and libraries using a Launchpad PPA and NVIDIA packages repository :
+
+.. code-block:: bash
+
+    sudo apt-key adv --fetch-keys "https://repo.download.nvidia.com/jetson/jetson-ota-public.asc"
+    sudo add-apt-repository -y "deb https://repo.download.nvidia.com/jetson/common r38.2 main"
+    sudo add-apt-repository -y "deb https://repo.download.nvidia.com/jetson/som r38.2 main"
+    # Install Tegra firmwares and necessary Nvidia libraries
+    sudo apt install -y nvidia-l4t-firmware nvidia-l4t-core nvidia-l4t-nvml nvidia-l4t-init
+    # Adding user to group render allows running GPU related commands as non root
+    # video group is necessary to use the camera
+    sudo usermod -a -G render,video ubuntu
+    sudo reboot
+
+Install CUDA and TensorRT
+=========================
+
+NVIDIA provides a few SDKs like CUDA Toolkit and TensorRT that allow building AI applications on Jetson devices. Those ones are maintained outside of Ubuntu but can be installed with the following commands:
```

Comment:
SDKs like CUDA Toolkit and TensorRT that allow building AI applications on Jetson devices are available directly from NVIDIA: